packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -378,8 +412,14 @@ class TextPainter {
   }
```

Comment:
add docs to layoutToMaxIntrinsicWidth saying that it overwrites previously configured constraint values.